roles/deploy_cr/tasks/main.yml

```diff
@@ -15,39 +15,38 @@
 - name: test_ Check for mandatory input  # noqa: name[casing]
   ansible.builtin.assert:
     that:
-      - api_version | string
-      - kind | string
-      - namespace | string
-      - name | string
-      - spec is mapping
+      - dc_api_version | string
+      - dc_kind | string
+      - dc_namespace | string
+      - dc_name | string
+      - dc_spec is mapping
```

Comment:
Could you please add argument_specs for these?

Reply:
done